scene/gui/text_edit.cpp

```diff
@@ -7404,8 +7404,8 @@ void TextEdit::_do_text_op(const TextOperation &p_op, bool p_reverse) {
 	}
 
 	if (insert) {
-		int check_line;
-		int check_column;
+		int check_line = -1;
+		int check_column = -1;
```

Comment:
GCC was complaining (now) that `check_line` and `check_column` could be uninitialized.

Reply:
Seems that my macro is more aggressive now, as it checks more thoroughly.

Reply:
Maybe it was only related to an error in my macro, I'll check if I revert all GCC related changes.